addons/website_sale_autocomplete/controllers/main.py

```diff
@@ -20,6 +20,17 @@
     'administrative_area_level_2': ['state', 'country']
 }
 
+CHILEAN_STATES = {
+    'Araucanía': 'de la Araucanía',
```

Comment:
in the XML there is a i without accent, so it doesn't seem that this one will work

```suggestion
    'Araucanía': 'de la Araucania',
```